node_app/models/Appointment.js

```diff
@@ -0,0 +1,77 @@
+/*
+Appointment {
+  status : enum (proposed, pending, booked, arrived, fulfilled, cancelled, noshow, entered-in-error, checked-in, waitlist)
+  participant : [ 
+    {
+      type : "patient" // note that this type is an extensible
+      actor : string "Patient/id"
+    },
+    {
+      type : "practitioner" // note that this type is an extensible
```

Comment:
We probably won't ever fill this in (the patient will just be registering for a bank of appointment openings for a given time slot), but it doesn't hurt to keep it.  I also think it's fine if you remove it.

Reply:
You're right. I'm not sure how appointments will be scheduled for practitioners and patients so I'll hold off on this for now.